README.md

```diff
@@ -214,7 +214,8 @@ The TZ (timezone) environment variable is set to 'Europe/London' in `start.js`.
 | REDIS_HOST **(required)** | Redis hostname. Can be used instead of `REDIS_URL`. Will override `REDIS_URL` if set | |
 | REDIS_AUTH_TOKEN | Optional auth token for the Redis instance | |
 | API_BASE_URL **(required)** | Base URL for the backend API server for this service without any path | `http://localhost:3000` |
-| API_PATH **(required)** | Base base for the API | `/api/v1` |
+| API_PATH **(required)** | Base path for the API | `/api` |
+| API_VERSION **(required)** | API version to use | |
```

Comment:
I noticed that an error gets through if this isn't set but wondered if we should set a default? And then it actually got me to thinking whether we should support this being set by an environment variable at all...because if we set it to the wrong version, would that not cause the rest of the application to break because it is built against a specific version?

So should this actually just be more of a temporary thing, like when we've introduced feature flags, to help us migrate to the new version, and then once that is done, we specifically set the version in the config?

Reply:
> So should this actually just be more of a temporary thing, like when we've introduced feature flags, to help us migrate to the new version, and then once that is done, we specifically set the version in the config?

Absolutely. This just gives us the flexibility to roll back instantly should the version change surface any unexpected issues.